src/Compilers/CSharp/Test/Symbol/Symbols/GenericConstraintTests.cs

```diff
@@ -2522,14 +2522,19 @@ class C<T>
 }";
             CreateCompilationWithMscorlib(source).VerifyDiagnostics(
                 // (4,15): error CS0246: The type or namespace name 'X' could not be found (are you missing a using directive or an assembly reference?)
+                //     where T : X
                 Diagnostic(ErrorCode.ERR_SingleTypeNameNotFound, "X").WithArguments("X").WithLocation(4, 15),
-                // (5,15): error CS0246: The type or namespace name 'I<T>' could not be found (are you missing a using directive or an assembly reference?)
-                Diagnostic(ErrorCode.ERR_SingleTypeNameNotFound, "I<T>").WithArguments("I<T>").WithLocation(5, 15),
+                // (5,15): error CS0246: The type or namespace name 'I' could not be found (are you missing a using directive or an assembly reference?)
```

Comment:
@AlekseyTs There is an issue #7493 for suggestions. `I<>` could work, but this might be confusing.